Client/Frontend/Browser/URLBarView.swift

```diff
@@ -146,6 +146,10 @@ class URLBarView: UIView {
         return locationView.active
     }
 
+    var canCancel: Bool {
+        return !cancelButton.hidden && cancelButton.enabled
```

Comment:
I thought about this too :-) seemed better for me to be sure. But as currently we do not disable the button anywhere, let's not confuse ourselves, so I just pushed a revised commit here that just checks `hidden` and not `enabled` (I also rebased).